src/command/test.rs

```diff
@@ -240,18 +246,6 @@ impl Test {
         Ok(())
     }
 
-    fn step_build_tests(&mut self, step: &Step) -> Result<(), Error> {
```

Comment:
Don't need to build the tests before running them since `cargo test` will build them so I removed it.

LMK if that is an issue